lib/ar_state_machine/configuration.rb

```diff
@@ -0,0 +1,11 @@
+module ARStateMachine
+  class Configuration
+
+    attr_accessor :system_id, :should_log_state_change
+
+    def initialize
+      self.should_log_state_change = true
```

Comment:
doing this gives me `No connection pool for ActiveRecord::Base` when I'm running tests.